rfcs/text/0000/threat.yml

```diff
@@ -0,0 +1,93 @@
+---
+- name: threat
+
+  fields:
+
+  - name: software.id
+    level: extended
+    type: keyword
+    short: Software ID assigned by MITRE ATT&CK.
+    description: >
+      Software ID assigned by MITRE for this code, OS utility, open source software, or tool used to conduct behavior modeled in ATT&CK.
```

Comment:
Doesn't need to be addressed for the stage 0 review, but there are conventions for MITRE ATT&CK® that we'll also want to follow in these descriptions.

You can see a list in https://github.com/elastic/ecs/pull/866.

Reply:
Cool. I made a few updates and we can tighten it up as we move forward.